packages/flutter_tools/lib/src/commands/run_mojo.dart

```diff
@@ -25,7 +29,11 @@ class RunMojoCommand extends FlutterCommand {
     argParser.addFlag('mojo-debug', negatable: false, help: 'Use Debug build of mojo');
     argParser.addFlag('mojo-release', negatable: false, help: 'Use Release build of mojo (default)');
 
-    argParser.addOption('app', defaultsTo: 'app.flx');
+    argParser.addOption('target',
+        defaultsTo: '',
```

Comment:
Defaulting to the empty string is effectively defaulting to the current working directory, which means defaulting to 'lib/main.dart'.